public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="kr">
```

Comment:
P2:
해당 페이지가 한국어로 되어 있다고 알려주시려면 `ko`로 적어주세요~

https://www.w3schools.com/tags/ref_language_codes.asp

```suggestion
<html lang="ko">
```